### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S CHANGELOG.md -L doesnt,optio,promisses,tread,usera`

### CODE REVIEW

File: packages/viewer/src/modules/SceneObjects.js [hunk 1]
```diff
@@ -295,7 +295,7 @@ export default class SceneObjects {
 
         let materialId = `${m.type}/${m.vertexColors}/${m.color.toJSON()}/${m.side}/${
           m.transparent
-        }/${m.opactiy}/${m.emissive}/${m.metalness}/${m.roughness}/${m.wireframe}`
+        }/${m.opacity}/${m.emissive}/${m.metalness}/${m.roughness}/${m.wireframe}`
```
Comment (luzpaz) [inline packages/viewer/src/modules/SceneObjects.js:298]: Please review closely

Comment (didimitrie) [reply packages/viewer/src/modules/SceneObjects.js:298]: It's a fun catch. I think it should be fine as these ids are transient/local session only relevant, but I'll ping @cristi8 to double check as it was his brain at work here :)

Comment (cristi8) [reply packages/viewer/src/modules/SceneObjects.js:298]: oh, yes, nice catch. this is actually a bug.. we would now group meshes with materials that only differ in opacity value, but have the same other attributes -> resulting in wrong materials for some objects in the final rendered scene

**Review by cristi8 — APPROVED** [review]

**Comment by didimitrie** [comment]
thanks @luzpaz, merged!

**Pull request merged**